src/harden.yml

```diff
@@ -15,15 +15,61 @@
         - https://raw.githubusercontent.com/cisagov/ansible-role-banner/develop/files/issue
         - https://raw.githubusercontent.com/cisagov/ansible-role-banner/develop/files/motd
     - name: Harden system
-      # This role is forked from konstruktoid/ansible-role-hardening
-      # and we do not control the names of the role variables.  This
+      # We do not control the names of the role variables.  This
       # is the reason for the noqa comment.
-      ansible.builtin.include_role:  # noqa var-naming[no-role-prefix]
-        name: harden
+      ansible.builtin.import_role:  # noqa var-naming[no-role-prefix]
+        name: konstruktoid.hardening
       vars:
-        # Point the role to the correct issue and motd templates
+        automatic_updates: true
+        fallback_ntp:
+          - 169.254.169.123
         issue_template: /tmp/issue
+        journald_storage: persistent
```

Comment:
```suggestion
        # Persist journald across reboots
        journald_storage: persistent
```

Reply:
We persist journald across reboots for of our AMIs, even those that are not hardened.  To remain consistent, perhaps we should continue to use our [cisagov/ansible-role-persist-journald](https://github.com/cisagov/ansible-role-persist-journald) even when using the [konstruktoid/ansible-role-hardening](https://github.com/konstruktoid/ansible-role-hardening) role.

Reply:
Using the [cisagov/ansible-role-persist-journald](https://github.com/cisagov/ansible-role-persist-journald) after the hardening role would ensure consistency but also add the  `journald_compress` and `journald_compress` variables if needed.